### PR TITLE
remove .bind(document) from elementsFromPoint

### PR DESCRIPTION
**Pull request by davidjmurphyjr**
…and treat elementsFromPoint as utility function instead of a polyfill

### CODE REVIEW

**Comment by yahoocla** [comment]
Thank you for submitting this pull request, however I do not see a valid CLA on file for you. Before we can merge this request please visit https://yahoocla.herokuapp.com/ and agree to the terms. Thanks! :smile:

File: src/Touch.js [hunk 1]
```diff
@@ -23,8 +23,7 @@ function getEventClientOffset (e) {
     }
 }
 
-// Polyfill for document.elementsFromPoint
-const elementsFromPoint = ((typeof document !== 'undefined' && document.elementsFromPoint) || function (x,y) {
```
Comment (JoaoMosmann) [inline src/Touch.js]: You removed the native `elementsFromPoint` when it's available. It should stay there for performance.

Just removing the bind might do the job.

**Comment by davidjmurphyjr** [comment]
Yup, you're totally right, it is a polyfill, I misunderstood what that first line was doing.

**Comment by JoaoMosmann** [comment]
You can run `yarn run dev` to test it locally.

**Comment by davidjmurphyjr** [comment]
Forgive my ignorance, but It seems like `yarn run dev` starts the dev server. With the dev server running, touch-dragging items around using chome dev tool's touch/responsive mode seemed to work fine. I tried both the index page and the /dropTarget page.  I wasn't able to find any automated/unit tests in the repo. Where can I go to see the broken tests?

**Comment by JoaoMosmann** [comment]
There aren't any automated tests at the moment. I'm planning on adding it in a near future, I became a collaborator here has less than a month, I'm trying to make the repo more reliable. If you want, help is always appreciated hehe.

When I said to run the dev, is to manually test if it's working the drag n' drop.

**Comment by davidjmurphyjr** [comment]
I'm down to help. If the repo gets more reliable, I could put it to good use. 

Do you think you could help me understand what removing bind breaks? I did not see any obvious issues in my manual testing, but that could be because i was testing on in Chrome's phone mode and not actually on a phone.

**Comment by JoaoMosmann** [comment]
Sorry for the delay. I don't recall the error but it wasn't working. Another guy did a PR that seems to work

**Pull request closed without merging**